index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="game.js"></script>
+    <title>Rock Paper Scissors Dashboard</title>
+</head>
+<body>
+    
+</body>
+</html>
```

Comment:
new lines are a must at end of files. Use `Prettier` to format your files, it takes care of such things